push/src/main/java/org/aerogear/mobile/push/PushService.java

```diff
@@ -394,7 +450,7 @@ public static void notifyHandlers(final Context context, final Map<String, Strin
      * </code>
      */
     @SuppressWarnings("unchecked")
-    private static void getDefaultHandler(final Context context) {
+    private static void setDefaultHandler(final Context context) {
```

Comment:
Now that I think about it, defaultHandler isn't a property so we probably shouldn't use "get" or "set".  Maybe initDefaultHandler, or createDefaultHangler?